subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AndroidTestProject.groovy

```diff
@@ -64,7 +64,7 @@ class IncrementalAndroidTestProject extends AndroidTestProject {
     static final SANTA_TRACKER_KOTLIN = new IncrementalAndroidTestProject(
         templateName: 'santaTrackerAndroidBuild',
         memory: '1g',
-        pathToChange: 'snowballrun/src/main/java/com/google/android/apps/santatracker/doodles/snowballrun/BackgroundActor.java',
+        pathToChange: 'common/src/main/java/com/google/android/apps/santatracker/AudioPlayer.kt',
```

Comment:
That won't work since we are using `ApplyNonAbiChangeToJavaSourceFileMutator` in `configureForNonAbiChange`, which can only change Java files.

Reply:
Damn you're right.

Reply:
Should be good now.